src/main/scala/uk/gov/nationalarchives/XMLCreator.scala

```diff
@@ -18,7 +18,7 @@ class XMLCreator {
         <Identifers>{if (isHierarchyFolder) <Identifer type="Code">{folder.name}</Identifer>}</Identifers>
       </opex:Properties>
       <opex:Transfer>
-        {if (isHierarchyFolder) <opex:SourceID>{folder.name}</opex:SourceID>}
+        {if (isHierarchyFolder && folder.`type` == ArchiveFolder) <opex:SourceID>{folder.name}</opex:SourceID>}
```

Comment:
We don't want to check title != name any more as it's possible now to have name != title for something that isn't a hierarchy folder. We've put the ArchiveFolder type in so we tell reliably.
You can change it so it's `val isHierarchyFolder: Boolean = folder.`type` == ArchiveFolder` and remove this addition here.